ml_model/src/main/java/edu/wm/sealab/featureextraction/Parser.java

```diff
@@ -114,6 +133,14 @@ public static void main(String[] args) {
                 double avgNumOfArithmeticOperators =
                     features.getArithmeticOperators() / entryNumLinesOfCode;
                 double avgNumOfConditionals = features.getConditionals() / entryNumLinesOfCode;
+                
+                double avgCommas = features.getCommas() / entryNumLinesOfCode;
+                double avgParenthesis = features.getParenthesis() / entryNumLinesOfCode;
+                double avgPeriods = features.getPeriods() / entryNumLinesOfCode;
+                double avgSpaces = features.getSpaces() / entryNumLinesOfCode;
+                double avgLineLength = features.getTotalLineLength() / entryNumLinesOfCode;
+                double avgIndentationLength = features.getTotalIndentation() / entryNumLinesOfCode;
+                double avgBlankLines = features.getTotalBlankLines() / entryNumLinesOfCode;
```

Comment:
For this I suggest instead of entryNumLinesOfCode, use blank_lines+entryNumLinesOfCode